### ISSUE TITLE
Any plans to add a mod to Modrinth?

### ISSUE THREAD

**Issue by Alter73** [florensie/swimmies#13]
Hi! I just make quite a lot of modpacks in the Modrinth launcher and a lot of mods have moved there. It is also possible because the service is developed only for Minecraft. It would be much more convenient if ExpandAbility library was also in Modrinth, otherwise I have to download it separately for each modpack and check for its updates, but this is very inconvenient when creating modpacks. It would be nice if such a cool library could be downloaded from the second most popular Minecraft modding site.

**Comment by florensie** [comment]
Is there any mod that doesn't already include it? Artifacts for example has Expandability embedded in its jar ;)

**Comment by Alter73** [comment]
Yes, but some older versions of mods, unfortunately, do not include it. And I've also seen some new mods.

**Comment by Kylas21** [comment]
Yes, it would be quite convenient if this mod was also in the Modrinth. After all, if someone creates a mod using this library, then it would be easy to add it as a dependency for a new mod.

**Comment by florensie** [comment]
Is there any particular version you need up on modrinth? I might not be bothered to upload all versions.

**Comment by Alter73** [comment]
I think the latest versions of NeoForge and Fabric for 1.20.4 and 1.20.1 are quite enough. Well, if it won't be very difficult for you, then please add the latest version for Forge and Fabric 1.19.4.

**Comment by Kylas21** [comment]
Yes, but it would also be nice to add the latest versions of the Fabric and Forge mod for 1.16.5. This version is very popular for modding.

**Comment by florensie** [comment]
Publishing to modrinth added in e26eb338989d39b848d4cc467094d998f5450b45, starting with v14.0.0. I will not be adding older versions.
